### ISSUE TITLE
Construct a generic autoplot function. @telkamp7

### ISSUE THREAD

**Issue by telkamp7** [ssi-dk/aedseo#12]
Construct a generic `autoplot()` function that can operate with `aedseo_tsd` and `aedseo` classes. Utilize the `ggplot2` package to make nice visualizations.

**Comment by telkamp7** [comment]
After #13 is merged into the codebase, it would be nice to also make a generic `autoplot()` function for that object.

**Comment by telkamp7** [comment]
Pull request #17 fixes this issue
